### DISCUSSION TITLE
几个小建议

### DISCUSSION THREAD

**Discussion by fcxqczzyjdxx** [tamlok/vnote#1859] [General]
建议把自动生成的vx_images vx_notebook  等文件夹改为隐藏形式...这样在笔记本列表中就不显示了,,建议把每个文档用到的图片单独建立一个文件夹,而不是公用的,如果多个文档公用的图片,可以设置是否复制,,建议从其他地方复制的图片的文章可以把图片复制过来,,比如从word复制的文章 比如从网站上复制的文章,,,,,,类似于typora对带图片文档的处理功能
在gitee上不知道怎么发讨论,所以在gitee上给您发了私信

**Comment by tamlok** [comment]
1. 隐藏的文件不会被第三方服务如坚果云默认同步，造成困扰；
2. 暂时没有；
3. vnote可以复制图片的，多探索一下右键菜单：
![image](https://user-images.githubusercontent.com/5763157/131235126-62bad0dd-4a69-4c37-bf77-67ee84be1681.png)

**Comment by morecup** [comment]
同2的需求，建议image的存储命名为 文件名+ .assets(这也是很多md编辑器的做法)，筛选时根据后缀不显示

**Reply by Fxiao1** [reply]
感觉“图片单独分文件夹”的需求不是太强烈，现在的图片都交由vnote管理了，无论是导出还是删除，vnote都会连同所引用的图片一同操作。好像也没有多少需要手动去管理的图片吧。如果想找到文档里面所有的图片的话也可以用正则筛选出来（ `!\[[^\]]*?\]\([a-z_]{1,2}_images/\d{17}_\d+\.png[^\)]*?\)` ）
